io.github.carterbox.Tofu.yml

```diff
@@ -0,0 +1,31 @@
+app-id: io.github.carterbox.Tofu
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+command: tofu
+modules:
+  - name: TOfU
+    buildsystem: simple
+    only-arches: x86_64
+    build-commands:
+      - mkdir -p /app/share/applications
+      - mv -vn io.github.carterbox.Tofu.desktop /app/share/applications/
+      - mkdir -p /app/share/metainfo
+      - cp -r io.github.carterbox.Tofu.metainfo.xml /app/share/metainfo/
+      - mkdir -p /app/share/icons/hicolor/512x512/apps
+      - cp -v icon.png /app/share/icons/hicolor/512x512/apps/io.github.carterbox.Tofu.png
+      - mkdir -p /app/Tofu
+      - mv -vn * /app/Tofu/
+      - chmod +x /app/Tofu/tofu
+      - mkdir -p /app/bin
+      - ln -s /app/Tofu/tofu /app/bin/tofu
+    sources:
+      - type: archive
+        url: https://github.com/carterbox/tofu/releases/download/1.0.4/tofu-linux-x86_64-1.0.4.tar.gz
```

Comment:
you are still build an aarch64 flatpak using x86_64 binaries. why no building from source?

Reply:
actually with no binaries, but that doesn't change the problem.

Reply:
I included `only-arches: x86_64`. Does that not restrict this flatpak to x86 only?

I'm not building for ARM because I am using the Flutter framework which does not support cross-compiling and I don't own any Linux ARM devices to test.

I'm not building this flatpak from source because to flatpak tooling doesn't seem to be tailored toward Flutter projects. It was much easier to build on GitHub actions where there exist actions already to setup Flutter in the build environment. AFAIK, flatpak doesn't have this type of composition mechanism because all flatpaks are meant to run in their own containers.

Reply:
Is there a way that I can contribute a Flutter build environment (All of Flutter is not needed at runtime) for flatpak that is reusable for other builders of Flutter apps?

Reply:
https://github.com/flathub/flathub/wiki/App-Maintenance#flathubjson